test/aws-config/mzcompose.py

```diff
@@ -0,0 +1,282 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import random
+import sys
+import time
+from dataclasses import dataclass
+
+import boto3
+from mypy_boto3_iam import IAMClient
+from mypy_boto3_sts import STSClient
+
+from materialize.mzcompose import Composition, UIError, WorkflowArgumentParser
+from materialize.mzcompose.services import Materialized, Testdrive
+
+MZ = Materialized()
+MZ.config["environment"] = [
+    e for e in MZ.config["environment"] if not e.startswith("AWS")
+]
+TESTDRIVE = Testdrive(
+    materialized_url=f"postgres://materialize@{MZ.name}:6875",
+)
+SERVICES = [
+    MZ,
+    TESTDRIVE,
+]
+
+SEED = str(random.randrange(0, 2 ** 32))
```

Comment:
I need access to _a_ seed here. I could extend the `Testdrive` service to expose the created seed, but that seems like it would make the Testdrive service definition different to all others.

I could also make the seeds unrelated, but this specific seed would still need to exist.

Reply:
I can also just use `seed=int` directly.